### PR TITLE
Migrated urls to path according to Django 3.2

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Addresses #1070

### CODE REVIEW

**Comment by gorkemarslan** [comment]
@AbhijithGanesh, I converted to draft because `urls.py` in `core` app includes `frontend_view`. The first step must be to move files in `frontend_view` to the related apps (`accounts`, and `threads`). Then we don't have to refactor `frontend_views` again and again.

I'm now preparing a guideline for `frontend_view` and self-assigned to it. It won't take much time. Then we can look at this PR. We just need to follow the order of merging PRs in order not to conflict.

Another thing we have to do is waiting for @brylie to take their approval for the issues. Then we can be ready for creating PR.

Edit:
Or first we can start frontend_view after merging this PR.

**Comment by AbhijithGanesh** [comment]
I think we can merge this PR and then work on frontend_views because I don't think it's gonna mess with our development.

**Comment by AbhijithGanesh** [comment]
@gorkemarslan Can you please work on frontend views to facilitate faster migration(I know speed is not a criterion) and I'm assigning #1073 to you to avoid confusion amongst members.

**Review by brylie — APPROVED** [review]

**Pull request merged**
